### ISSUE TITLE
Start Problem since latest commit

### ISSUE THREAD

**Issue by IMikkeI** [wnameless/docker-oracle-xe-11g#12]
Hi,
since the new commit, the container doesn't start anymore. Is there a known issue with it ?

Thanks

**Comment by wnameless** [comment]
I just merged a pull request few days age. I haven't tested it yet. Give me some time, if there is a fatal error; I will revert it to previous version immediately.

**Comment by IMikkeI** [comment]
Ok thank you

**Comment by wnameless** [comment]
I just made a test. Everything goes well on my laptop. Can you give me more details about your issue?

**Comment by IMikkeI** [comment]
I have pulled docker-oracle-xe-11g image.
When I run the command  "docker run -d -p 49160:22 -p 49161:1521 wnameless/oracle-xe-11g" and I check with docker ps, the container isn't running. If I run the image with -it instead of -d, I can see the following error :

Starting Oracle Net Listener.
Starting Oracle Database 11g Express Edition instance.
Failed to start Oracle Net Listener using /u01/app/oracle/product/11.2.0/xe/bin/tnslsnr and Oracle Express Database using /u01/app/oracle/product/11.2.0/xe/bin/sqlplus.

I'm am using Docker on Ubuntu:14.04.

**Comment by wnameless** [comment]
Could you please check the creation date of your image?
It should be 5 days ago and 2.241gb in size.

**Comment by IMikkeI** [comment]
Yes they're exactly alike.

**Comment by wnameless** [comment]
Did you stop and remove the previous container?

**Comment by IMikkeI** [comment]
Yes. I use Ubuntu on a virtual machine. I have a base Ubuntu with a fresh install of Docker and just pulled your image yesterday. Nothing is running (docker ps) and I have also suppressed all old containers (docker ps -aq is empty)

**Comment by sebhoss** [comment]
i have a similar issue on OSX:

```
is-6:tmp shoss$ docker ps
CONTAINER ID        IMAGE                     COMMAND                CREATED             STATUS              PORTS                                                      NAMES
7b867e01a44d        wnameless/oracle-xe-11g   "/bin/sh -c '/usr/sb   2 minutes ago       Up 2 minutes        8080/tcp, 0.0.0.0:49160->22/tcp, 0.0.0.0:49161->1521/tcp   evil_darwin
```

looks like it started normally, but i can't connect to the database on port 49161 (or any other port). Image was pulled today.

Tried running it with docker-compose as well, but get the following:

```
Attaching to tmp_oracle_1
oracle_1 | Starting Oracle Net Listener.
oracle_1 | Starting Oracle Database 11g Express Edition instance.
oracle_1 |
```

So again it seems like the database is up, but i still can't connect

**Comment by IMikkeI** [comment]
Have you an error like "got minus one from a read call"  when you try to connect ?

**Comment by sebhoss** [comment]
SQL Developer tells me

> Status : Failure - Test failed: IO Error: The Network Adapter could not establish the connection

firewalls are deactivated, running latest docker, latest boot2docker and similar containers for mongodb/postgres/redis/elasticsearch are working fine here.

**Comment by IMikkeI** [comment]
When it worked for me (I have tested oracle/docker on windows and now linux), I use a command line like this : 

docker run -d -p IP:49161:1521 -p IP:49160:22 wnameless/docker-oracle-xe-11g 
where IP is the IP you got with boot2docker ip

After I connect with sqldeveloper with this IP instead of localhost. 
Try to do it and tell me, if it can helps you it will be great.

**Comment by sebhoss** [comment]
oh yeah you are right: been running docker on linux for a while and did basically forget about boot2docker. In my case everything works fine if you replace localhost with the whatever `boot2docker ip` gives you or you can port-forward from the boo2docker VM to the real host with this:

```
# if the VM is not yet running
VBoxManage modifyvm "boot2docker-vm" --natpf1 "tcp-port49161,tcp,,49161,,49161";

# if the VM is already running
VBoxManage controlvm "boot2docker-vm" natpf1 "tcp-port49161,tcp,,49161,,49161";
```

after that connecting with the settings specified in the README of this project works fine. See [the boot2docker doc](https://github.com/boot2docker/boot2docker/blob/master/doc/WORKAROUNDS.md#port-forwarding) for work arounds on OSX

**Comment by IMikkeI** [comment]
I've seen the solution with VBoxManage but I tried without success.

Is the last commit of wnameless/docker-oracle-xe-11g image works on your computer on a linux ?

**Comment by sebhoss** [comment]
yes it runs fine on ubuntu 14.04 with both docker/compose 1.6/1.2 and 1.7/1.3.

**Comment by IMikkeI** [comment]
Ok.
I don't understand why I have the "Failed to start Oracle Net Listener" when I try to run it :(

**Comment by ovanekem** [comment]
I had same problem. The issue was that my boot2docker VM was running out of disk space.
Just to make sure you have the same problem as I have, just start your container using:
```
docker run -it wnameless/oracle-xe-11g /bin/bash
```

You'll end in a bash, then type:
```
df -h
```

If the Filesystem shows 0 available for the Filesystem "none", then you have the same issue.
Basically I just dropped the disk and recreated with more space as explained here: https://docs.docker.com/articles/b2d_volume_resize/

Then redo the same operation, when you are certain that you have sufficient disk space, start oracle inside your container:
```
/usr/sbin/startup.sh
```

After it started, doing a:
```
ps -eaf
```
should return the list of Oracle processes running (xe_pmon_XE,...). 

I figured that afterwards, doing this, then closing the container, then restarting,... a couple of time seems that some hard disk space is leaked, I don't know if others experience the same ?
